lib/storage/file.js

```diff
@@ -1528,7 +1528,7 @@ File.prototype.startResumableUpload_ = function(dup, options) {
     metadata: {}
   }, options);
 
-  var uploadStream = resumableUpload({
+  var config = {
```

Comment:
Feel free to just add the uri from the `options` object. It will be ignored if falsy from gcs-resumable-upload.

```js
  ...
  uri: options.uri
});
```